test/v2/controllers/V2DeparturesControllerSpec.scala

```diff
@@ -1083,6 +1115,99 @@ class V2DeparturesControllerSpec
     }
   }
 
+  // Version 2
+  "for a departure update with accept header set to application/vnd.hmrc.2.0+json (version two)" - {
```

Comment:
Can you please make this resemble the other test descriptions in the spec?

Reply:
done